pnpm-lock.yaml

```diff
@@ -6384,6 +6384,10 @@ packages:
     engines: {node: '>=18.0.0', npm: '>=8.0.0'}
     hasBin: true
 
+  rosetta@1.1.0:
```

Comment:
Could we discard changes in this file? We aren't changing anything related with deps